<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1072</width>
    <height>704</height>
   </rect>
  </property>
  <property name="baseSize">
   <size>
    <width>1080</width>
    <height>720</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>微信读书助手-wereader</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resource.qrc">
    <normaloff>:/favicon.ico</normaloff>:/favicon.ico</iconset>
  </property>
  <property name="documentMode">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="browser">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="baseSize">
    <size>
     <width>1080</width>
     <height>700</height>
    </size>
   </property>
   <property name="autoFillBackground">
    <bool>true</bool>
   </property>
   <widget class="QSplitter" name="splitter_2">
    <property name="geometry">
     <rect>
      <x>5</x>
      <y>0</y>
      <width>1070</width>
      <height>700</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
    <widget class="QSplitter" name="splitter">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <widget class="QListView" name="listView"/>
     <widget class="QTableView" name="tableView"/>
    </widget>
    <widget class="QTextEdit" name="noteEdit"/>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1072</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuTool">
    <property name="title">
     <string>工具</string>
    </property>
    <addaction name="actionLoadShelf"/>
    <addaction name="separator"/>
    <addaction name="actionLoadNotes"/>
    <addaction name="actionLoadCover"/>
   </widget>
   <widget class="QMenu" name="menuNote">
    <property name="title">
     <string>笔记</string>
    </property>
    <addaction name="separator"/>
    <addaction name="actionShow"/>
    <addaction name="actionSave"/>
    <addaction name="separator"/>
    <addaction name="actionLoadHot"/>
    <addaction name="actionOpen"/>
   </widget>
   <widget class="QMenu" name="menuAbout">
    <property name="title">
     <string>关于</string>
    </property>
    <addaction name="actionAuthor"/>
    <addaction name="actionLicense"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>视图</string>
    </property>
    <addaction name="actionback"/>
    <addaction name="actionforward"/>
    <addaction name="separator"/>
    <addaction name="actionShelf"/>
    <addaction name="actionLibrary"/>
    <addaction name="separator"/>
   </widget>
   <addaction name="menuNote"/>
   <addaction name="menuTool"/>
   <addaction name="menuView"/>
   <addaction name="menuAbout"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionLoadNotes">
   <property name="text">
    <string>下载全部笔记</string>
   </property>
  </action>
  <action name="actionShow">
   <property name="text">
    <string>切换笔记模式</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Tab</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>保存笔记</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionAuthor">
   <property name="text">
    <string>Author &lt;arry_lee@qq.com&gt;</string>
   </property>
   <property name="toolTip">
    <string>Author</string>
   </property>
  </action>
  <action name="actionLicense">
   <property name="text">
    <string>License</string>
   </property>
  </action>
  <action name="actionLoadShelf">
   <property name="text">
    <string>加载书架</string>
   </property>
   <property name="toolTip">
    <string>Download shelf</string>
   </property>
  </action>
  <action name="actionLoadHot">
   <property name="text">
    <string>热门笔记</string>
   </property>
  </action>
  <action name="actionLoadCover">
   <property name="text">
    <string>下载封面</string>
   </property>
  </action>
  <action name="actionback">
   <property name="text">
    <string>后退</string>
   </property>
   <property name="toolTip">
    <string>后退</string>
   </property>
   <property name="shortcut">
    <string>B</string>
   </property>
  </action>
  <action name="actionforward">
   <property name="text">
    <string>前进</string>
   </property>
   <property name="toolTip">
    <string>前进</string>
   </property>
   <property name="shortcut">
    <string>F</string>
   </property>
  </action>
  <action name="actionShelf">
   <property name="text">
    <string>书架</string>
   </property>
   <property name="shortcut">
    <string>S</string>
   </property>
  </action>
  <action name="actionLibrary">
   <property name="text">
    <string>书城</string>
   </property>
   <property name="shortcut">
    <string>L</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="text">
    <string>打开笔记</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="resource.qrc"/>
 </resources>
 <connections/>
</ui>
